ament_lint_auto/cmake/ament_lint_auto_find_test_dependencies.cmake

```diff
@@ -30,11 +31,28 @@ macro(ament_lint_auto_find_test_dependencies)
     ament_package_xml()
   endif()
 
+  set(EXCLUDED ${ARG_EXCLUDE})
+  foreach(_exclusion ${ARG_EXCLUDE})
+    # If a package has already been found, we don't want to
+    # set <package>_FOUND to FALSE at the end
+    if(${_exclusion}_FOUND)
+      list(REMOVE_ITEM EXCLUDED ${_exclusion})
+    else()
+      set(${_exclusion}_FOUND TRUE)
+    endif()
+  endforeach()
```

Comment:
Instead of faking the `FOUND` flag here why not skip `_dep`s in the following `foreach` if they are listed in `ARG_EXCLUDE`?

Reply:
@dirk-thomas Skipping `find_package` only works if the package is a direct dependency (and is therefore in `*_TEST_DEPENDS`); however, if you are using a meta package like `ament_lint_common`, `find_package(ament_cmake_cppcheck)` will not be skipped